app/js/instances/instanceModel.js

```diff
@@ -103,12 +103,34 @@ function instanceModel(dockerService) {
                 });
```

Comment:
So this `alertService.create()` is what's causing @zach-taylor's issue with the django pull. The reason this won't work but the `dockerService.d('image.list', ...)` will is that the docker service has a noticeable round-trip time - long enough for the initial flux action to finish dispatching before the result comes back in. But, since the alerts are local, it interrupts the call stack.

One option is to set some kind of state flag in the model here so the listener can pick up on it, and then fire off the alert.